src/main/java/com/aws/greengrass/telemetry/impl/TelemetryLoggerMessage.java

```diff
@@ -28,7 +29,7 @@ public String getJSONMessage() {
         try {
             return OBJECT_MAPPER.writeValueAsString(this.metricDataPoint);
```

Comment:
Modify the objectmapper in this class to so that it doesn't fail. We may just want a shared mapper for both.

Reply:
Added. Maybe keep them separate for now in case they have different serialization requirements.